src/sentry/static/sentry/app/components/dateTime.jsx

```diff
@@ -54,7 +54,9 @@ const DateTime = React.createClass({
       date = new Date(date);
     }
 
-    return <time {...carriedProps}>{moment(date).format(format)}</time>;
+    return (
+      <time {...carriedProps}>{moment.tz(date, options.timezome).format(format)}</time>
```

Comment:
How are we currently handling timezones?

Reply:
to be honest, I don't understand. I know we allow users to set a timezone preference but i'm not sure how that gets used